ansible/ca-cert.yml

```diff
@@ -0,0 +1,27 @@
+# An ansible playbook to configure the SSHD configuration to enable CA cert auth for SSH.
+# Remember to export CI_CLOUD if it isn't part of your environment's variables.
```

Comment:
Why is this needed? How are you running this playbook?

Reply:
Yeah we need to talk about where this is included in site.yml, there's a bigger job/discussion here than just adding a playbook I think.